ia-guide/ia-static.md

```diff
@@ -0,0 +1,341 @@
+This will help you know which templates to use from the Good Docs Project, and give you the basics to start completing them.
+
+If you want to do this using the interactive tool, go here instead: [Interactive IA Guide](don't click here the link does not exist yet).
+The interactive version is the same as this static version, so you can use this version to check back on later on if you need to.
+
+## What are you writing about?
+
+The first thing to do is think about what you're writing about.
+
+Are you writing about something pretty simple, that people can use without having any experience? Like a toaster?
+
+Or are you writing about something a bit complicated, that people can start to use only after they've done a bit of reading, and might need to ask some questions as they go along? More like a car?
+
+Or are you writing about something really tricky? Something that requires training, a whole lot of background knowledge, and possibly a friend to help them the whole way? Like a nuclear reactor?
+
+**Curse of Knowledge alert!!**
+> Try to remember that not everyone knows the same stuff, or learns in the same way.
+> What if your reader doesn't speak the same language, or is a new migrant from a place with a totally different culture?
+> What if they are nine years old?
+> Or 99 years old?
+> What if they have a physical or mental disability?
+> When you are thinking about your readers, make sure you are thinking about everyone, not just people like you.
+
+### Writing about a toaster
+
+Because the thing you are writing about is pretty simple and straightforward, you might think that you don't need much documentation.
+
+Just because you might not need a lot of docs, you will still need something, and working that out could be tricky.
+You're going to need to think a little bit harder about who you're writing for.
+
+![A silver toaster marked "Super Tooster" with a red discount star, and a sign saying "wowzars!!!"](images/tooster.svg "Super Tooster")
+
+### Writing about a car
+
+There is a pretty good chance that you needed to learn how to do this thing yourself, and hopefully it wasn't all that long ago, and you can fairly easily cast your mind back to those days when you didn't know what this thing was, or how it worked.
+And hopefully there is also someone nearby who has had to learn as well, so go ask them questions!
+
+![A cheering person in a green outfit driving a red and white striped sports car, with the caption "big red car"](images/car.svg "Sports car")
+
+### Writing about a nuclear reactor
+
+The first thing to work out if you are writing about a nuclear reactor is what reasonable assumptions can you make?
+There's a fair chance that you can assume at least some kind of prior knowledge: a degree, some relevant experience, or perhaps using some other related tools or software.
+
```

Comment:
More concrete types of docs

- I guarantee there are 3-ring binders complete with operational How-Tos / Runbooks on how to Triage and Resolve monitoring errors/warnings happening across the various system.  
- There are probably documented policies on what to do when certain alerts/warnings happen -- some may even have legal requirements in term of reporting them to government agencies.
- Not to mention the outstanding Reference documents.